Gemfile

```diff
@@ -13,17 +13,10 @@ gemspec
 
 gem "rspec", ">=3.0"
 gem "rspec-rails"
-gem "sqlite3", "~> 1.6.0.rc2"
-gem "sidekiq", "~> 6.1", ">= 6.1.2"
+gem "sqlite3"
 gem "standard"
-gem "appraisal"
+gem "sidekiq", "~> 7.0"
```

Comment:
I'm not sure I like this, because Sidekiq is now too tangled with acts_as_tenant. In fact, the sidekiq integration, DelayedJob or others, should be separate gems that depend on this one

IMO sidekiq doesn't belong in the Gemfile and should be in it's own appraisal

Reply:
This is simply for testing and there is no actual dependency on Sidekiq.